src/components/board/board.jsx

```diff
@@ -1,19 +1,20 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types'
+
+import React from 'react';
+import PropTypes from 'prop-types';
 import { Container, Row } from 'react-bootstrap';
+
 import Square from '../square';
 
```

Comment:
Try keeping a single empty line between imports here so that the file doesn't become unnecessarily long